src/_includes/layouts/event.html

```diff
@@ -29,9 +33,47 @@
 		<div data-layout="centered-single-column">
 			<main class="events-page">
 				<h1>{{ title }}</h1>
-				<small> {% if speakers %} {{ speakers | join: ', ' }} &middot; {% endif %} {{ date }} </small>
+				<small> {% if speakers %} {{ speakers | join: ', ' }} &middot; {% endif %} {{ date | asDateTime }} </small>
 				{{ addToCalendarList }} {{ content }}
-				<p>Join this event <a href="https://discord.com/invite/q9ZdBgP">in Discord</a></p>
+				<div>
```

Comment:
@BenDMyers good question!

I have a Draft PR that addresses what that near-future might look like.
Thoughts on the eventual approach? https://github.com/LunchDevCommunity/community-calendar/pull/44

Reply:
I dig that a lot!

What do _you_ think about the approach?

Reply:
i feel good!

i like it from the perspective of having a single reusable element.

i'm torn on the exact approach (as mentioned in a recent comment on that draft).
with my React history, i have a natural pull to toward this being an element that i can further customize. partials and shortcakes don't really provide the same level of extensibility.

that said — for this exact button — i think any of the options are "good enough™️"

Reply:
applied to this specific cluster of elements, I might hold off further extracting them until we have reason to use them on other pages. simply to avoid extracting the wrong thing and making future work more painful.